istio/istio.io
# Network topology automated test (#9096)

Manual cherry-pick of https://github.com/istio/istio.io/pull/9096 to include changes necessary for it to work with 1.9 release branch


[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure
* Cleanup make snips issues

* Fix snippets

* Add test for network topologies page

* Check output; but don't be too strict

gateway_url works differently under metallb than it does on a provisioned k8s cluster

* Try verify_likes

* Fix unit test

* Get the automated test to work with verify_like

* Fix snippet

* Fix test failures

* Use verify like instead of contains

* gateways work differently on kind than elsewhere
